packages/messages/src/browser/notifications-message-client.ts

```diff
@@ -48,21 +52,59 @@ export class NotificationsMessageClient extends MessageClient {
     }
 
     showProgress(progressId: string, message: ProgressMessage, cancellationToken: CancellationToken, update?: ProgressUpdate): Promise<string | undefined> {
-        const messageArguments = { ...message, type: MessageType.Progress, options: { ...(message.options || {}), timeout: 0 } };
-        if (this.visibleProgressNotifications.has(progressId)) {
+        const messageArguments = {
```

Comment:
why should the default value for `timeout` be changed now?

Reply:
Because I understand the problem - In the case when you call all notifications from the status bar widget, it is not enough time to read.
You can close it manually with the 'close' button.
We are using default timeout the first time only

Reply:
https://github.com/theia-ide/theia/blob/4382a27f709c60cacfe1113534fda0f2e7d0d80e/packages/messages/src/browser/notifications-message-client.ts#L160

Reply:
if we remove ```, timeout: 0```  it will get it from  preferences['notification.timeout']

Reply:
got it. the question is, why should that be the case in general? what is wrong about it in general? besides the git clone operation, the main use case when introducing this feature was to allow progress reporting. what happens with that now?

if the source for progress reports stops updating, will this dismiss the progress notification? 

if you think this should be a short timeout value for git clone in this PR, why not just setting it explicitly?

Reply:
I cannot understand you.  Previously it was shown without a close timeout at all ``` timeout: 0 ```. 

If the source for progress reports stops updating(as I understand - cancel this progress notification
https://github.com/theia-ide/theia/pull/4483/files#diff-cde0e8b29563cef99910b9e41f4daab7R89
), will this dismiss the progress notification.